en/guide/using-middleware.md

```diff
@@ -150,6 +151,7 @@ Load router-level middleware by using the `router.use()` and `router.METHOD()` f
 The following example code replicates the middleware system that is shown above for application-level middleware, by using router-level middleware:
 
 ```js
+var express = require('express')
```

Comment:
there is one more section just below this section where `var express = require('express')` is missing. Could you pls add that as well?

Reply:
Yes, I have created a commit for changes. Thanks